src/microbe_stage/organelle_components/MovementComponent.cs

```diff
@@ -58,12 +65,16 @@ protected override void CustomAttach()
     /// <summary>
     ///   Calculate the momentum of the movement organelle based on
     ///   angle towards middle of cell
+    ///   If the flagella is  placed in the microbe's center, hence delta equals 0,
+    ///   consider defaultPos as the organelles "false" position.
     /// </summary>
-    private static Vector3 CalculateForce(Hex pos, float momentum)
+    private static Vector3 CalculateForce(Hex pos, float momentum, Vector3 defaultPos)
     {
-        Vector3 organelle = Hex.AxialToCartesian(pos);
+        Vector3 organellePosition = Hex.AxialToCartesian(pos);
         Vector3 middle = Hex.AxialToCartesian(new Hex(0, 0));
```

Comment:
It might be worth having a constant for Hex(0, 0) in the dedicated class. There are some already, but none for this very one.

Reply:
I thinkt that should be in a diff PR, this PR's goal is different.